base/gmp.jl

```diff
@@ -514,8 +514,7 @@ hex(n::BigInt) = base(16, n)
 function base(b::Integer, n::BigInt)
     2 <= b <= 62 || throw(ArgumentError("base must be 2 ≤ base ≤ 62, got $b"))
     p = ccall((:__gmpz_get_str,:libgmp), Ptr{UInt8}, (Ptr{UInt8}, Cint, Ptr{BigInt}), C_NULL, b, &n)
-    len = Int(ccall(:strlen, Csize_t, (Cstring,), p))
-    String(pointer_to_array(p,len,true))
+    unsafe_string_wrapper(p, true)
```

Comment:
`unsafe_string_wrapper` calls `strlen` if you don't pass `len`.   Anyway, `mpz_get_str` returns a nul-terminated string, so it won't have embedded nuls.